src/js/ripple/uint160.js

```diff
@@ -90,9 +92,9 @@ UInt160.prototype.to_json = function(opts) {
       }
 
       return output;
-    } else {
-      return this.to_hex();
     }
+    // else
```

Comment:
Will clean up scraggles tomoz then. Unless someone else wants to... ? What
a joyous thing to wake up to :)